src/hooks/useUMIP.ts

```diff
@@ -0,0 +1,39 @@
+import { useQuery } from "react-query";
+import * as contentful from "contentful";
+
+export default function useUMIP(number?: number) {
+  const { data, ...others } = useQuery<UMIP>(
+    ["umip", number],
+    () => fetchUmip(number!),
+    {
+      enabled: number != null,
+    }
+  );
+
+  return { umip: data, ...others };
+}
+
+const contenfulSpaceId = process.env.REACT_APP_PUBLIC_CONTENTFUL_SPACE_ID;
+const contentfulAccessToken =
+  process.env.REACT_APP_PUBLIC_CONTENTFUL_ACCESS_TOKEN;
+
+const contentfulClient = contentful.createClient({
+  space: contenfulSpaceId || "",
+  accessToken: contentfulAccessToken || "",
```

Comment:
i think contentful should be optional, especially during dev/staging.  this is technically optional, but things might break having empty strings here.  i have a suggestion for this let me know what u think

Reply:
Could you elaborate on why you think it should be optional? I think we would want to develop in an env as close as possible to the production one, to catch things as early as possible.
My only worry is if something breaks on the contenful integration the we would find out in prod if we don't use in development

Reply:
well for this particular app, i dont see how we can do this since we wont always be testing on mainnet. how would contentful work on a testnet or kovan or something like that?

Reply:
Oh I see what you mean now, Contentful would still work, but the ids in the CMS would not match the actual requests done in the `dev` environment. For example if you added a proposal `Admin 100` on Kovan and the CMS had data for UMIP 100 then it would show the UMIP 100 description in the `dev` environment, conversely, if the CMS had no data for UMIP 100 then it would just show the fallback description